CHANGELOG.txt

```diff
@@ -122,6 +122,20 @@
                     - In 'models.calendar' :
                         - The method 'CalendarManager.new_color()' is deprecated.
                         - The method 'Calendar.get_color()' is deprecated; use 'Calendar.color' instead.
+                * Sketch:
+                    - Add 'creme.d3NumericDataInfo' that create stats from data : min, max, only integers, value range, average.
+                    - Add 'creme.d3NumericFormat' & 'creme.d3NumericAxisFormat' to find the best format from data information.
+                    - Add 'creme.d3Scroll' renderer to handle horizontal or vertical scrolling of an svg node
+                    - creme.D3BarChart: Horizontal scrolling if a minimum bar width is reached.
+                    - creme.D3DonutChart:
+                        - Vertical scrolling of legend on overflow.
+                        - Slice text is visible only on mouse hover if it is too small.
+                    - Fix an issue in 'creme.d3LegendRow' renderer visible in D3TubeChart (legend without labels).
+                    - creme.d3ColorRange now accept a function. Useful for computed ranges.
+                * Reports:
+                    - creme.D3TubeChart:
+                        - Horizontal scrollog of legend & textwrap.
+                        - Bar text is visible only on mouse hover if it is too small.
```

Comment:
A small explanation of the improvements in the "users side" part would be better.
scrollog => scrolling I suppose.